src/components/navigation/AppNavigationBoard.vue

```diff
@@ -34,39 +34,65 @@
 			style="opacity: 0.5" />
 
 		<template v-if="!deleted" slot="actions">
-			<ActionButton v-if="canManage && !board.archived"
+			<ActionButton v-if="!showDueSettings"
+				icon="icon-more"
+				:close-after-click="true"
+				@click="actionDetails">
+				{{ t('deck', 'Board details') }}
+			</ActionButton>
+			<ActionButton v-if="canManage && !board.archived && !showDueSettings"
 				icon="icon-rename"
 				:close-after-click="true"
 				@click="actionEdit">
 				{{ t('deck', 'Edit board') }}
 			</ActionButton>
-			<ActionButton v-if="canManage && !board.archived"
+			<ActionButton v-if="canManage && !board.archived && !showDueSettings"
```

Comment:
This is :see_no_evil:

Reply:
:see_no_evil: Will clean up once the sub-menu is somewhat approved by @jancborchardt ;)

Reply:
Refactored the submenu handling with the last commit